RELEASES.md

```diff
@@ -29,9 +30,12 @@
 - `taffy::error::InvalidNode` has been removed and is now just a branch on the `TaffyError` enum
 - `taffy::forest::Forest` has been merged into `taffy::node::Taffy` for a performance boost up to 90%
 - `Size::undefined()` has been removed, use `Size::NONE` instead.
+- `Taffy.compute_layout()` now takes `Size<AvailableSpace>` instead of `Size<Option<f32>>`
+- Measure functions now have an additional `available_space` parameter which indicates the size of the parent or a min/max-content sizing constraint.
```

Comment:
```suggestion
- Measure functions now have an additional `available_space` parameter which indicates the size of the parent or a min/max-content sizing constraint.
- `Size::NONE` has been renamed to `Size::MAX_CONTENT` for clarity.
```

Reply:
@alice-i-cecile That's not quite right. Size::NONE still exists (and still produces a `Size<Option<f32>>`). `Size::MAX_CONTENT` (and Size::MIN_CONTENT`) are additional and produce a `Size<AvailableSpace>`.

Reply:
```suggestion
- Measure functions now have an additional `available_space` parameter which indicates the size of the parent or a min/max-content sizing constraint.
- Added `Size::MIN_CONTENT` and `Size::MAX_CONTENT` constants. In many cases, you will want to replace `Size::NONE` with `Size::MAX_CONTENT`.
```

Does that work better?

Reply:
Sure, that works :)